polling_stations/apps/data_finder/features/index.feature

```diff
@@ -1,14 +1,24 @@
 Feature: Check Postcodes
-    Scenario: Check Islington
-    Given I put in the postcode "N19SN"
```

Comment:
Didn't realise that you were explicitly checking that pollingstations will work for me personally on every test run!